src/pages/LandingPage/LandingPage.jsx

```diff
@@ -15,6 +15,16 @@ const LandingPage = () => (
         <h2>It's all in the details</h2>
       </div>
     </div>
+    <div
+        id="about-section"
+        className="w-full flex bg-cover bg-right"
```

Comment:
I think this should probably be something like `bg-center-bottom` (lock to bottom middle of image) so the car doesn't get too small on ultrawide.

Reply:
You're right, but with this the car doesn't get displayed in some views, changed to `bg-center`.

Reply:
Visually I would have preferred `bg-bottom` but this works significantly better for responsiveness so that's alright